src/main/java/Constants.java

```diff
@@ -0,0 +1,101 @@
+public class Constants {
+    public static final String introPrint = "----------------------------------------------------------------------------------\n"
+            + "    BMO chop!\n    Do you want to play video games?\n"
+            + "----------------------------------------------------------------------------------\n";
+    public static final String tutorialPrint = "Command BMO with these keywords!\n"
+            + "0. hi [greet BMO]\n"
+            + "1. bye [shut BMO down]\n"
+            + "2. log [view task log]\n"
+            + "3. add todo <task> [add todo task]\n"
+            + "4. add deadline <task> /by <date> <time> [add deadline]\n"
+            + "5. add event <event> /from <date> <time> /to <date> <time> [add event]\n"
+            + "6. done <task number> [check task as done]\n"
+            + "7. redo <task number> [uncheck task]\n"
+            + "----------------------------------------------------------------------------------\n";
+
+    public static final String hiPrint = "----------------------------------------------------------------------------------\n"
+            + "    Good day! What can BMO help you with?\n"
+            + "----------------------------------------------------------------------------------\n";
+
+    public static final String byePrint = "----------------------------------------------------------------------------------\n"
+            + "    Beep boop BMO shutting down...\n"
+            + "----------------------------------------------------------------------------------\n";
+
+    public static final String emptyLogPrint = "----------------------------------------------------------------------------------\n"
+            + "    Wow! Your log is actually empty.\n"
+            + "    Let's play mario kart right now!!\n"
+            + "----------------------------------------------------------------------------------\n";
+
+    public static class errorPrint {
```

Comment:
maybe you can consider renaming the class to errorPrints, since it has multiple error messages